hurricane/management/commands/consume.py

```diff
@@ -40,17 +42,30 @@ def add_arguments(self, parser):
         )
         parser.add_argument("handler", type=str, help="The Hurricane AMQP handler class (dotted path)")
         parser.add_argument(
-            "--probe",
+            "--liveness-probe",
             type=str,
             default="/alive",
-            help="The exposed path (default is /alive) for probes to check liveness and readyness",
+            help="The exposed path (default is /alive) for probes to check liveness",
+        )
+        parser.add_argument(
+            "--readiness-probe",
+            type=str,
+            default="/ready",
+            help="The exposed path (default is /ready) for probes to check readyness",
+        )
+        parser.add_argument(
+            "--startup-probe",
+            type=str,
+            default="/startup",
+            help="The exposed path (default is /startup) for probes to check startup",
         )
         parser.add_argument(
             "--probe-port",
             type=int,
             default=8001,
             help="The port for Tornado probe route to listen on",
         )
+        parser.add_argument("--req-queue-len", type=int, default=10, help="Lenght of the request queue")
```

Comment:
> Looks good to me. Should we have further tests for the three probes? Current tests only include the startup-probe explicitly.
Right now tests also include readiness and liveness probe testing, but they are implicitly defined in the startup-probe-tests. I think we can just rename the tests so that they do not become misleading.